### ISSUE TITLE
airDatepickerInput off by one day

### ISSUE THREAD

**Issue by jlbarr** [dreamRs/shinyWidgets#99]
If i set a value for the input, the value shown is off by a day.   example value='2018-03-15'.  On the web page I would see 2018-03-14.  demoAirDatepicker("update") does the same thing.  The value shown and the value on the slider are off by a day.

**Comment by pvictor** [comment]
Mmmh probably a timezone issue, what's your locale ?

```r
# in R
Sys.timezone()
# [1] "Europe/Berlin"

# in Javascript console
Date()
# "Thu Jul 26 2018 16:33:05 GMT+0200 (heure d’été d’Europe centrale)"
```

**Comment by jlbarr** [comment]
EDT - Eastern Daylight Time

**Comment by lemairev** [comment]
thank you, if I can help I would be happy (but I doubt I can)
cheers
vincent

**Comment by pvictor** [comment]
I made some changes, if you can test it'll be nice.

You need to re-install from Github first.

```r
devtools::install_github("dreamRs/shinyWidgets")
```

You can try this example and yours : 

```r
library(shiny)
library(shinyWidgets)

ui <- fluidPage(
  airDatepickerInput(
    inputId = "multiple",
    label = "Select multiple dates:", value = "2018-09-15",
    placeholder = "You can pick 5 dates",
    multiple = 5, clearButton = TRUE
  ),
  verbatimTextOutput("res_multiple")
)

server <- function(input, output, session) {
  output$res_multiple <- renderPrint(input$multiple)
  
}

shinyApp(ui, server)
```

**Comment by lemairev** [comment]
I've reinstalled without problem, but I face the same issue with both example: 2018-09-14 is the starting day in the calendar for me with your testapp.

Would you like that I give you more information? Like my R version...

**Comment by madmark81** [comment]
Running it in google chrome with yesterday's version worked without problem
for me.
If it changes I'll let you guys know. Perhaps update your r

**Comment by lemairev** [comment]
ok ! I work on fedora28, R.3.5.0 and use google chrome ! 
If the problem is solved for the other at least it's great :) 
thank you
cheers

**Comment by pvictor** [comment]
That's a timezone issue in the browser.
Does the dates returned by the widget are correct ? E.g. if you select `2018-09-10` in the calendar, the input value is `2018-09-10` ?
![image](https://user-images.githubusercontent.com/4415580/45610286-bbc76f80-ba5b-11e8-8f83-9ce845a9e88a.png)

**Comment by lemairev** [comment]
yes you are probably right ! I"ll try to fix this, thank you for your help :)

**Comment by lemairev** [comment]
Ok problem solved guys ! thank you
I read date (yyyymmdd) but when it is turned to posixct (probably in the picker) it messed up
eg. i read 20170701 when it turned to posix it's 20170701:00 + timezone 
==> 20100630:20 for me 
I have to take that into account now !
thank you again, and thank you for that great package :)

**Comment by jlbarr** [comment]
Tried it today and read the date out of the picker correctly, but when I attempt to set it's value, it set the date one day back.  i.e. 2019-03-01 become 2019-02-28.

**Comment by slodge** [comment]
We've encountered this problem recently.

We've not fully debugged it but it feels like dates coming from R (from the server) need to have an offset applied.

See this example JavaScript from a desktop sitting in SF:

![image](https://user-images.githubusercontent.com/533662/100879326-36e19880-34a3-11eb-8a7d-bbf8a13d16ac.png)

One fix might be to apply offsets to all the inputs in e.g. `initialize` and  `setValue` -e.g. lines like: https://github.com/dreamRs/shinyWidgets/blob/ced9b19315f83ff3add17e22eb05a0dcf3d99d96/inst/assets/air-datepicker2/datepicker-bindings.js#L19 ?

**Comment by pvictor** [comment]
Can you try to install from #323 ?

```r
remotes::install_github("dreamRs/shinyWidgets#323")
```

and see if it helps ?
